tests/migration/migration.go

```diff
@@ -2949,6 +2949,12 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 			return false
 		}
 
+		runPod := func(pod *k8sv1.Pod) *k8sv1.Pod {
+			pod, err := libpod.Run(pausePod, testsuite.GetTestNamespace(pod))
```

Comment:
Oops right, getting `pausePod` from the scope is a mistake on my side. I'll include the error handling in the loop. IMO this lambda could be justified but since it's only used in one loop I'm fine changing it.